cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/DatasetServiceClient.java

```diff
@@ -271,13 +271,12 @@ private HttpResponse doDelete(String resource) throws DatasetManagementException
     return doRequest(HttpMethod.DELETE, resource);
   }
 
-  private HttpResponse doRequest(HttpMethod method, String resource,
-                                 @Nullable Multimap<String, String> headers,
-                                 @Nullable String body) throws DatasetManagementException {
+  private HttpResponse doRequest(HttpMethod method, String resource, String body,
```

Comment:
I saw that in this particular helper method, the ``body`` was wrongly marked ``@Nullable``. So I removed it. Then changed the order to keep all nullable parameters at the end. Can revert this change though.